azkaban-common/src/main/java/azkaban/imagemgmt/daos/ImageVersionDaoImpl.java

```diff
@@ -65,34 +84,49 @@ public int createImageVersion(ImageVersion imageVersion) {
       imageVersionId = databaseOperator.transaction(insertAndGetSpaceId).intValue();
     } catch (SQLException e) {
       log.error("Unable to create the image version metadata", e);
+      throw  new ImageMgmtDaoException("Exception while creating image version metadata");
```

Comment:
Nit: Can you please format the code using Azkaban checkstyle? You can set it up using steps mentioned in https://github.com/azkaban/azkaban/blob/master/CONTRIBUTING.md

Reply:
Code is already formatted using AzkabanStyle codestyle based on https://iwww.corp.linkedin.com/wiki/cf/display/ENGS/New+hire%27s+guide+to+Azkaban+setup

Reply:
Throw and new has two spaces in between. Can you please format it once again?
If you will configure save action like mentioned in contributing.md, new line will get added in the end. A lot of other things are also mentioned in it.